assignment2/task2.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: UTF-8 -*-
+
+import math
+
+
+class LCABuilder(object):
```

Comment:
For better understanding this I explain all the algorithm.

On this task we should find least common ancestor (lca) of two vertexes. I made this with binary lift method.